### ISSUE TITLE
modify of zoom/position on filtered map

### ISSUE THREAD

**Issue by josefspr** [daat-hamakom/maps#154]
selection of timeline range used to work in the past, but doesn't work now.
same should be done for map.

**Comment by mushon** [comment]
@josefspr what do you mean? Is that click and drag to select time range? (click and drag is used for panning the timeline) or did you mean something else?

**Comment by josefspr** [comment]
No, I just mean that the zoom and position of the map & timeline change when you make a search
according to events

**Comment by mushon** [comment]
right, @yuvadm that is indeed missing and should also be supported by the browser's [Back] button
